webapp/src/components/DynamicDataTableButtons.vue

```diff
@@ -270,6 +279,11 @@ export default {
     columnLabel(option) {
       return option.label || option.header || option.field;
     },
+    resetTable() {
+      if (window.confirm("Are you sure you want to completely reset the Datatable?")) {
```

Comment:
```suggestion
      if (window.confirm("Are you sure you want to reset your preferences (visible columns, widths) for this table?")) {
```

Can we make this slightly clearer?